ui/i18n/src/main/res/values/strings.xml

```diff
@@ -320,6 +320,10 @@
     <string name="confirm_mobile_streaming_notification_message">Streaming over mobile data connection is disabled in the settings. Tap to stream anyway.</string>
     <string name="confirm_mobile_streaming_button_always">Always</string>
     <string name="confirm_mobile_streaming_button_once">Once</string>
+    <string name="download_started_talkback">%1$s download started</string>
```

Comment:
Wouldn't `Started download of %1$s` be more natural? It would also make it more clear what happened if the title is long

Reply:
I'm not sure about natural, but it's definitely more clear. I'll adjust that.

Reply:
I went with 'Download <action> <episode title>', I think that hierarchy makes it even more usable.

Reply:
Maybe `Download <action> of/for <episode title>`

Reply:
This is current setup:

```
<string name="download_started_talkback">Download started for %1$s</string>
<string name="download_completed_talkback">Download completed for %1$s</string>
```

As far as I can tell "of" wouldn't be appropriate when maintaining current order.